package.json

```diff
@@ -117,8 +117,8 @@
     "postcss-loader": "2.0.10",
     "prop-types": "15.6.0",
     "query-string": "^5.1.1",
-    "react": "16.2.0",
-    "react-dom": "16.2.1",
+    "react": "^16.14.0",
```

Comment:
Do we need the `^` since the next version is 17 and it's already out?

Reply:
I think its common to keep updating older versions with security patches, etc